sdk/cosmos/azure-cosmos/samples/concurrency_sample.py

```diff
@@ -0,0 +1,106 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+
+import os
+from azure.cosmos import PartitionKey, ThroughputProperties
+from azure.cosmos.aio import CosmosClient
+import asyncio
+import time
+
+# Specify information to connect to the client.
+CLEAR_DATABASE = True
+CONN_STR = os.environ['CONN_STR']
+# Specify information for Database and container.
+DB_ID = "Cosmos_Concurrency_DB"
+CONT_ID = "Cosmos_Concurrency_Cont"
+# specify partition key for the container
+pk = PartitionKey(path="/id")
+
+# Batch the creation of items for better optimization on performance.
```

Comment:
This isn't really a batch though? It's just calling the `create_item` method many times concurrently?
Given that you're API actually supports batching (and that features is in beta), I think it's a bit confusing to refer to this as a "batch".
Additionally, it might be even more interesting to use the transactional batch format as an additional comparison here.

Reply:
Adding transactional batch would be interesting, but the current concurrency sample can run any other operation concurrently as it does with point operations. Adding it might not show a significant difference to what we currently have.

Reply:
I do agree that using the word batch might confuse some users, what would you suggest as an alternative? We could try a more verbose approach and replace batching with "Create 100 items per concurrent call"